client/src/main/java/com/alibaba/nacos/client/env/NProperties.java

```diff
@@ -16,12 +16,17 @@
 
 package com.alibaba.nacos.client.env;
 
+import java.util.Properties;
+
 /**
- * nacos env interface.
+ * NProperties interface.
  *
  * @author onewe
  */
-public interface NacosEnvironment {
+@SuppressWarnings("PMD.ClassNamingShouldBeCamelRule")
+public interface NProperties {
```

Comment:
What's the meanings of `N`? Nacos or other?

Reply:
The `N` means `Nacos`

Reply:
Or change from the `NProperties` to another names? I have no idea what name is better.

Reply:
Just Nacos client properties？

Reply:
Yes? it's only client properties currently. What else does it have any properties?